pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -28,8 +38,12 @@ mkdir -p '{{ $unit.DropIns.Path }}'
 
 systemctl daemon-reload
 {{ if .Bootstrap }}
+{{ if isContainerDEnabled .CRI -}}
+systemctl enable containerd && systemctl restart containerd
+{{- else -}}
 systemctl enable docker && systemctl restart docker
 {{ end -}}
```

Comment:
```suggestion
{{ end }}
```

Reply:
This was causing an incorrect systemctl statement. It was directly attached to the next statement like so `systemctl enable containerd && systemctl restart containerdsystemctl restart.....`.